sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/Models/CognitiveServicesAccountApiProperties.cs

```diff
@@ -40,12 +40,24 @@ public CognitiveServicesAccountApiProperties()
         /// flag to enable statistics of Bing Search.</param>
         /// <param name="storageAccountConnectionString">(Personalization Only)
         /// The storage account connection string.</param>
-        public CognitiveServicesAccountApiProperties(string qnaRuntimeEndpoint = default(string), bool? statisticsEnabled = default(bool?), string eventHubConnectionString = default(string), string storageAccountConnectionString = default(string))
+        /// <param name="aadClientId">(Metrics Advisor Only) The Azure AD
```

Comment:
Editing this file by hand is not allowed.  Please see : https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec  for how to regenerate your adk from an updated swagger.  You will also need to ensure the merged swagger PR is referenced in the comments for your PR

Reply:
Hi mark, this is auto generated. I didn't check-in all changes though.